### DISCUSSION TITLE
caching the instance?

### DISCUSSION THREAD

**Discussion by dhallX** [PhilippMolitor/react-unity-renderer#27] [Q&A]
is it possible to cache the module instance with the package? I need the unity component to render once and then be saved for the rest of the session so that when the user comes back to the page they don't rerender the unity project?

**Comment by PhilippMolitor** [comment]
As long as you don't change the props on the renderer component (which you usually wouldn't do), it does not rerender the unity instance. If you need to make modifications to the viewport of the game, you should wrap the renderer in a styling component, and make the renderer fill it via width/height 100%.
